multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/BuildCloudDeployModelStep.java

```diff
@@ -64,12 +64,13 @@ protected StepPhase executeStep(ProcessContext context) {
         // Get module sets:
         DeployedMta deployedMta = context.getVariable(Variables.DEPLOYED_MTA);
         List<DeployedMtaApplication> deployedApplications = (deployedMta != null) ? deployedMta.getApplications() : Collections.emptyList();
-        Set<String> mtaArchiveModules = context.getVariable(Variables.MTA_ARCHIVE_MODULES);
-        getStepLogger().debug(Messages.MTA_ARCHIVE_MODULES, mtaArchiveModules);
+        Set<String> mtaManifestModulesNames = context.getVariable(Variables.MTA_ARCHIVE_MODULES);
```

Comment:
What is the difference between manifest and descriptor modules? Is it related with MANIFEST.MF and mtad.yaml ?

Reply:
If it is a partial deploy, the manifest.mf contains only the specified modules for deployment (as well as the mtar).